rllib/tuned_examples/dqn/cartpole-apex.yaml

```diff
@@ -0,0 +1,26 @@
+cartpole-apex-dqn:
```

Comment:
should you report the results of this tuned example in the comments at top of file?

Reply:
^^

Reply:
bump on this @sven1977 ?

Reply:
We can do that, but usually - for these simple tasks - the stop criteria define what should happen: Learn in 200k steps to get to 100 reward.